lib/tasks/gitlab/check.rake

```diff
@@ -278,6 +279,29 @@ namespace :gitlab do
       end
     end
 
+    def check_upload_perms
+      print "Upload directory permissions correct? ... "
+
+      upload_path = Rails.root.join(Gitlab.config.gitlab.user_home,"gitlab-rails","uploads")
+      upload_path_tmp = Rails.root.join(upload_path,"tmp")
+
+      # if tmp upload dir has incorrect permissions, assume others do as well
+      if File.stat(upload_path_tmp).mode == 16877 && File.owned?(upload_path_tmp) # verify drwxr-xr-x permissions
+        puts "yes".green
+      else
+        puts "no".red
+        try_fixing_it(
+          "sudo chown -R #{gitlab_user} #{upload_path}",
```

Comment:
The `upload_path` is owned by group specified at https://github.com/gitlabhq/omnibus-gitlab/blob/master/files/gitlab-config-template/gitlab.rb.template#L307 from what I can tell. I'm just not sure how to access that variable/where it is in non-omnibus.